src/imp/util/util.js

```diff
@@ -9,6 +8,11 @@ class Util {
         }
         return timer;
     }
+
+    shouldSendMetaSpan(opts, tags) {
+        let shouldSendSpan = opts.meta_event_reporting === true && !tags['lightstep.meta_event'] === true;
```

Comment:
I'm wondering if `tags['...'] !== true` is more readable than `!tags['...'] === true`

Reply:
Hm, maybe, yeah. I'll update it.